firmware/lib/globalTime/globalTime.cpp

```diff
@@ -127,16 +127,25 @@ bool GlobalTime::isPM() {
 
 void GlobalTime::getTimeZoneOffsetFromAPI() {
     HTTPClient http;
-    http.begin(String(TIMEZONE_API_URL) + "?key=" + TIMEZONE_API_KEY + "&format=json&fields=gmtOffset&by=zone&zone=" + String(TIMEZONE_API_LOCATION));
```

Comment:
Rather than removing the fields so we get logs of data we should be able to just add the new field we want to the field list

Reply:
I would be good to merge this as is but this tweak would be an improvement

Reply:
The API reply is so small even without filtering that it does not really make a difference, but I've readded the "fields" parameter.